src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function onSuccess() {
+  document.body.insertAdjacentHTML('beforeend', `<div class='message'>
+    Promise was resolved!
+  </div>`);
+}
+
+function onError() {
+  document.body.insertAdjacentHTML('beforeend',
+    `<div class='message error-message'>
+      Promise was rejected!
+    </div>`
+  );
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+promise1.then(
+  value => onSuccess(),
+  error => onError(error)
+);
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
```

Comment:
Don't disable the eslint, but fix its problem.
Try googling the error text to understand it